app/src/main/java/de/westnordost/streetcomplete/quests/trail_visibility/AddTrailVisibility.kt

```diff
@@ -0,0 +1,35 @@
+package de.westnordost.streetcomplete.quests.trail_visibility
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddTrailVisibility : OsmFilterQuestType<TrailVisibility>() {
+
+    override val elementFilter = """
+        ways with
+          highway ~ path|footway|cycleway|bridleway
+          and !trail_visibility
+          and (sac_scale and sac_scale != hiking)
```

Comment:
```suggestion
          and (sac_scale and sac_scale != hiking)
          and (!lit or lit = no)
```

if it is lit (in any way), its visibility is `excellent`, so no need to ask...

Reply:
> But that would expect all paths to have a 'lit' tag of some kind

Ummm, no? `!lit` (unless I'm mistaken) would match on all ways that do not have `lit` tag mapped (regardless of its value). So the quest would be asked either if there is no `lit` tag at all, _**or**_ if there is explicit `lit=no`.
(but would not be asked on `lit=yes`, `lit=24/7`, `lit=automatic`, etc.)